website/routes.py

```diff
@@ -191,14 +191,19 @@ def make_url_map(app):
 
     process_rules(app, [
 
-        Rule('/dashboard/', 'get', website_views.dashboard, OsfWebRenderer('dashboard.mako')),
+        Rule('/dashboard/', 'get', website_views.redirect_to_home, OsfWebRenderer('home.mako')),
+        Rule('/myprojects/', 'get', website_views.dashboard, OsfWebRenderer('dashboard.mako')),
```

Comment:
Ping @JeffSpies for route review.

- `/myprojects/` -> project organizer; linked to in navbar

![osf___my_projects](https://cloud.githubusercontent.com/assets/2379650/13652661/6d966fd0-e61b-11e5-90ef-b867db6fa95e.png)

- `/` when logged in -> user home, containing "My projects" and "New and noteworthy nodes"; linked to in the top-left of navbar

![osf___home](https://cloud.githubusercontent.com/assets/2379650/13652699/a60847d0-e61b-11e5-8e85-886b97fa5599.png)

- `/dashboard/` -> redirects to `/` ; included for backwards compat
- `/support/` -> New support page

![osf___support](https://cloud.githubusercontent.com/assets/2379650/13652754/f49aafc8-e61b-11e5-95f6-6cb49924a205.png)

@erinmayhood and I are good with these changes.